minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -174,29 +151,18 @@ public void start(TachyonConf tachyonConf) throws IOException {
     mMasterConf.set(Constants.USER_DEFAULT_BLOCK_SIZE_BYTE, Integer.toString(mUserBlockSize));
     mMasterConf.set(Constants.USER_REMOTE_READ_BUFFER_SIZE_BYTE, "64");
 
-    // Since tests are always running on a single host keep the resolution timeout low as otherwise
-    // people running with strange network configurations will see very slow tests
-    mMasterConf.set(Constants.HOST_RESOLUTION_TIMEOUT_MS, "250");
-
-    // Lower the number of threads that the cluster will spin off.
-    // default thread overhead is too much.
-    mMasterConf.set(Constants.MASTER_MIN_WORKER_THREADS, "1");
```

Comment:
There is already duplicate set/code in LocalTachyonMaster(final String tachyonHome, TachyonConf tachyonConf) in minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java. These sets are a little bit confusing, since there are distributed in two methods....which may need further merge of these two files.